module/move/gspread/src/commands/gspread.rs

```diff
@@ -22,35 +22,41 @@ mod private
   #[ derive( Debug, Parser ) ]
   pub struct CommonArgs
   {
-    #[ arg( long ) ]
+    #[ arg( long, help = "Full URL of Google Sheet.\n\
+    It has to be inside of '' to avoid parse errors.\n\
+    Example: 'https://docs.google.com/spreadsheets/d/your_spreadsheet_id/edit?gid=0#gid=0'" ) ]
     pub url : String,
 
-    #[ arg( long ) ]
+    #[ arg( long, help = "Sheet name.\nExample: List1" ) ]
```

Comment:
```suggestion
    #[ arg( long, help = "Sheet name.\nExample: Sheet1" ) ]
```

This is better for an example because it is the default name that is always generated.